lib/pattern-importer.js

```diff
@@ -0,0 +1,155 @@
+var through = require('through2'),
+  utility = require('./utility'),
+  merge = require('lodash.merge'),
+  gutil = require('gulp-util'),
+  path = require('path'),
+  fs = require('fs'),
+  PluginError = gutil.PluginError,
+  PLUGIN_NAME = 'pattern-importer';
+
+/*
+ * Imports html patterns that are following the standards at github.com/pattern-library
+ *
+ * @param {Object} options
+*/
+exports.patternImporter = function (options) {
```

Comment:
@e2tha-e This repository is never used directly and always called from `./node_modules` as an imported module. Making the path this way was the only consistent way I found to have this file work.

I'd love to find another way to include this - not sure what that is though as this works consistently.